### PR TITLE
re comment pd service yaml value

### PR DESCRIPTION
**Pull request by mikechengwei**
# What is changed and how does it work?
Original pd clusterIP is none, it will be parsed "None" string. It's an error.
So I comment pd service config，and remind users of this feature.
### Check List 
Tests 
 - Unit test

### CODE REVIEW

File: charts/tidb-cluster/values.yaml [hunk 1]
```diff
@@ -76,7 +76,10 @@ pd:
   # pd Service
   # we can only specify clusterIP and loadBalancerIP now
   service:
```
Comment (weekface) [inline charts/tidb-cluster/values.yaml]:
```suggestion
  service: {}
```

?

Comment (mikechengwei) [reply charts/tidb-cluster/values.yaml]: Thanks, I have sloved

**Comment by cofyc** [comment]
ah, it conflicts with another PR you created, you need to resolve it.

**Comment by mikechengwei** [comment]
ok.

**Pull request merged**